portfolio/src/main/webapp/style.css

```diff
@@ -1,23 +1,59 @@
-#content {
+#content{
   margin: 20px;
 }
 
-#nav-bar {
+#nav-bar{
   display: flex;
   flex-direction: row;
 }
 
-#scroll-buttons {
+.prim-color{
+  color: #fca61f;
+}
+
+#scroll-buttons{
   margin-left: auto;
 }
 
-.flex-hor {
+.flex-hor{
   display: flex;
   flex-direction: row;
   justify-content: space-between;
 }
 
-button {
+.flex-col{
+  display: flex;
+  flex-direction: column;
+}
+
+form{
+  margin: auto;
+  max-width: 70em;
+}
+
+textarea{
```

Comment:
[4.2.5](https://google.github.io/styleguide/htmlcssguide.html#Property_Name_Stops): Use a space between the last selector and the declaration block.